pkg/probe/crd_probe.go

```diff
@@ -84,12 +99,12 @@ func (r *ReadinessCRD) Check(_ *http.Request) error {
 }
 
 // IsReady returns true if all required CRDs are ready
-func (r *ReadinessCRD) IsReady() (bool, error) {
+func (r *CRDProbe) IsReady() (bool, error) {
 	return r.check()
 }
 
-func (r *ReadinessCRD) check() (bool, error) {
-	list, err := r.client.ApiextensionsV1beta1().CustomResourceDefinitions().List(v1.ListOptions{})
+func (r *CRDProbe) check() (bool, error) {
+	list, err := r.client.ApiextensionsV1beta1().CustomResourceDefinitions().List(v1.ListOptions{LabelSelector: "svcat=true"})
```

Comment:
please replace with 

```go
labels.SelectorFromSet(labels.Set{
		"svcat": "true",
}).String()
```

I do not want to hardcode the `=` sign here. It is an implementation details, so better will be to use helpers:)